_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,260 @@
+---
+title:Recommendations Report 
+date: 2019-08-02
+authors:
+  - Service Innovation Lab
+tags:
+  - blog post
+  - open source
+  - rates rebates
+  - Te Whakamāmā i ngā Reiti
+excerpt: "Following a successful Alpha phase in May and September 2018, the Rates Rebate beta trial was run in June and July 2019 by a team with members from Service Delivery and Operations team within DIA, Government Information Services, and the Service Innovation Lab. This was in collaboration with Tauranga City Council, Hutt City Council, and Kāpiti Coast District Council"
+---
+# Executive Summary
```

Comment:
You need a space after a header. And you're only allowed one top level header (which should describe the whole doc, not just a segment.

So let's drop this header down, by using two `#`


```suggestion
## Executive Summary

```